src/Overlay.js

```diff
@@ -65,34 +66,50 @@ const propTypes = {
 };
 
 const defaultProps = {
-  animation: Fade,
+  transition: Fade,
   rootClose: false,
   show: false,
-  placement: 'right',
+  placement: 'top',
 };
 
-class Overlay extends React.Component {
-  render() {
-    const { animation, children, ...props } = this.props;
+function wrapRefs(popper) {
+  const {
+    ref,
+    arrowProps: { ref: aRef },
+  } = popper;
+  popper.ref = ref.__wrapped || (ref.__wrapped = r => ref(findDOMNode(r)));
```

Comment:
could use `Symbol` here... we have `runtime`

Reply:
runtime in v7 no longer polyfills by default, there is like a whole separate runtime now for corejs polyfills